laravel/tests/Unit/EvaluationParserTest.php

```diff
@@ -8,22 +8,53 @@
 
 class EvaluationParserTest extends TestCase
 {
-    /**
-     * A basic test example.
-     *
-     * @return void
-     */
     use DatabaseTransactions;
+
+    public function getPrivateMethod( $className, $methodName ) {
+		$reflector = new \ReflectionClass( $className );
+		$method = $reflector->getMethod( $methodName );
+		$method->setAccessible( true );
+		return $method;
+	}
```

Comment:
Let's include a brief explanation of what this is for & a link to where we found this code.